opm/autodiff/SimulatorFullyImplicitBlackoilOutput.cpp

```diff
@@ -441,7 +390,11 @@ namespace Opm
                 restorefile >> state;
                 restorefile >> wellState;
 
-                writeTimeStep( timer, state, wellState );
+                // FIXME: We this should optimally have the proper per cell data to dump
```

Comment:
Is this not exactly the same behaviour as we have upon initialization? If so, I don't think this needs the FIXME comment.